ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -1091,30 +1079,30 @@ gst_tensor_src_tizensensor_create (GstBaseSrc * src, guint64 offset,
   return retval;
 }
 
-#define cast_loop (values, count, dest, desttype) \
+#define cast_loop(values, count, dest, desttype) \
 do { \
   int i; \
   char *destptr = (char *) (dest); \
   for (i = 0; i < (count); i++) \
     *(destptr + (sizeof (desttype) * i)) = (desttype) (values)[i]; \
 } while (0)
 
-#define case_cast_loop (values, count, dest, desttype, desttypeenum) \
+#define case_cast_loop(values, count, dest, desttype, desttypeenum) \
 case desttypeenum: \
   cast_loop (values, count, dest, desttype); \
   break;
 
 /**
  * @brief Copy sensor's values[] to Gst memory map.
  */
-void
-_ts_assign_values (float values[], int count, GstMapInfo *map,
-    const GstTensorInfo *spec)
+static void
+_ts_assign_values (float values[], int count, GstMapInfo * map,
+    const GstTensorInfo * spec)
 {
-    switch (spec->type) {
-      case _NNS_FLOAT32:
-        mamcpy (map->data, values, sizeof(float) * count);
-        break;
+  switch (spec->type) {
+    case _NNS_FLOAT32:
+      memcpy (map->data, values, sizeof (float) * count);
+      break;
       case_cast_loop (values, count, map->data, int64_t, _NNS_INT64);
```

Comment:
Need to shift `case_cast_loop()` to the left?

Reply:
gst-indent won't recognize it and emit error if I do that :(

Reply:
Oh... i see.